src/Board.js

```diff
@@ -74,6 +75,7 @@ class Board extends React.Component {
   };
 
   //reset the state so player can restart the game
+  // is this really a reset? I don't see all properties being reset here as per default state definition.
   reset = () => {
```

Comment:
if you fully reset to the default state, maybe it makes sense to define a variable called `defaultState`
E.g.:

```
const defaultState = {
     board: createBoard(),
     score: 0,
     ....
}
```